src/main/java/io/percy/selenium/Percy.java

```diff
@@ -190,9 +190,6 @@ public void screenshot(String name, Map<String, Object> options) throws Unsuppor
         List<String> driverArray = Arrays.asList(driver.getClass().toString().split("\\$")); // Added to handle testcase (mocked driver)
         Iterator<String> driverIterator = driverArray.iterator();
         String driverClass = driverIterator.next();
-        if (!driverClass.equals(RemoteWebDriver.class.toString())) { throw new UnsupportedOperationException(
```

Comment:
Optional:
Check `driverClass instanceof WebDriver` instead

Reply:
Not required because webdriver is the default type and intellij suggests the same.
<img width="774" alt="image" src="https://github.com/percy/percy-selenium-java/assets/29580782/2bb85319-fc67-447b-bac7-93a3d08c8d72">